v21.2/show-jobs.md

```diff
@@ -127,6 +130,45 @@ You can filter jobs by using `SHOW AUTOMATIC JOBS` as the data source for a [`SE
 (1 row)
 ~~~
 
+### Show changefeed jobs
+
+<span class="version-tag">New in v21.2:</span> You can display specific fields relating to changefeed jobs by running `SHOW CHANGEFEED JOBS`, which include:
+
+* [`high_water_timestamp`](stream-data-out-of-cockroachdb-using-changefeeds.html#monitor-a-changefeed): Guarantees all changes before or at this time have been emitted.
+* [`sink_uri`](create-changefeed.html#sink-uri): The destination URI of the configured sink for a changefeed.
+* `full_table_names`: Provides the full [name resolution](sql-name-resolution.html) for a table. For example, `defaultdb.public.mytable`, the database (`defaultdb`), schema (`public`), and table name (`mytable`).
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW CHANGEFEED JOBS;
+~~~
+
+~~~
+job_id               |                                                                                   description                                                                  | user_name | status  |              running_status              |          created           |          started           | finished |          modified          |      high_water_timestamp      | error |         sink_uri       |      full_table_names      | format
+---------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------+---------+------------------------------------------+----------------------------+----------------------------+----------+----------------------------+--------------------------------+-------+------------------------+----------------------------+---------
+685724608744325121   | CREATE CHANGEFEED FOR TABLE mytable INTO 'kafka://localhost:9092' WITH confluent_schema_registry = 'http://localhost:8081', format = 'avro', resolved, updated | root      | running | running: resolved=1629336943.183631090,0 | 2021-08-19 01:35:43.19592  | 2021-08-19 01:35:43.225445 | NULL     | 2021-08-19 01:35:43.252318 | 1629336943183631090.0000000000 |       | kafka://localhost:9092 | {defaultdb.public.mytable} | avro
+685723987509116929   | CREATE CHANGEFEED FOR TABLE mytable INTO 'kafka://localhost:9092' WITH confluent_schema_registry = 'http://localhost:8081', format = 'avro', resolved, updated | root      | paused  | NULL                                     | 2021-08-19 01:32:33.609989 | 2021-08-19 01:32:33.64293  | NULL     | 2021-08-19 01:35:44.224961 | NULL                           |       | kafka://localhost:9092 | {defaultdb.public.mytable} | avro
+(2 rows)
+~~~
+
+Changefeed jobs can be [paused](stream-data-out-of-cockroachdb-using-changefeeds.html#pause), [resumed](stream-data-out-of-cockroachdb-using-changefeeds.html#resume), and [canceled](stream-data-out-of-cockroachdb-using-changefeeds.html#cancel).
```

Comment:
I think in certain publications in the US they go for the double "l".... but I've heard people say that they are just being all lofty! I think about the double "l" a lot because I try not to let any BrE slip in :-)